xcfs/Sources/build/main.swift

```diff
@@ -0,0 +1,65 @@
+// use it from root folder:
+// `swift run --package-path xcfs build [all, awk, tar, ios_system, ...]`
+
+import Foundation
+import FMake 
+
+OutputLevel.default = .error
+
+// TODO: We can add more platforms here
+let platforms: [Platform] = [.iPhoneOS, .iPhoneSimulator]
+
+let allSchemes = [
+    "ios_system",
+
+    "awk",
+    "tar",
+    "curl_ios", 
+    "files",
+    "shell",
+    "ssh_cmd",
+    "tar",
+    "text",
+    ]
+
+let args = ProcessInfo.processInfo.arguments 
+
+var schemes: [String] 
+if args.count > 1 && args[1] != "all" {
```

Comment:
I recommend to use [swift-argument-parser](https://github.com/apple/swift-argument-parser).

Reply:
You recommend to replace 3 lines of code with another 3 lines of code + 1mb of code dependency.... 

I don't think it is worth it for now.

Reply:
You are right for now, but I think it's worth replacing for readability and extensibility of code.